tests/test_html2docx.py

```diff
@@ -29,6 +29,8 @@ def test_html2docx(html_path, spec_path):
     for p, p_spec in zip(doc.paragraphs, spec):
         assert p.text == p_spec["text"]
         assert p.style.name == p_spec.get("style", "Normal")
+        if p.alignment:
+            assert int(p.alignment) == p_spec.get("alignment", 0)
```

Comment:
Instead of defaulting to 0, can force the test to specify the expected result.
```suggestion
            assert int(p.alignment) == p_spec["alignment"]
```